Dockerfile

```
@@ -0,0 +1,15 @@
+# syntax=docker/dockerfile:1
+FROM golang:1.18.1
```

If it fits your use-case, I suggest to take the alpine version which is lighter
```suggestion
FROM golang:1.18.1-alpine
```
See weight difference by browsing https://hub.docker.com/_/golang?tab=tags&page=2

---

Yes i'm gonna go with alpine version, it doesn't seem to make a change on the execution